### ISSUE TITLE
windows 10 above the font exception

### ISSUE THREAD

**Issue by typ431127** [amueller/word_cloud#306]
My environment is windows 10 python2.7 But I run the script will be the wrong font
```
  File "C:\Python27\lib\site-packages\PIL\ImageFont.py", line 143, in __init__
    self.font = core.getfont(font, size, index, encoding, layout_engine=layout_engine)
IOError: cannot open resource
```
font='C://Windows/Fonts/Fontssimhei.ttf'
I tried using some fonts do not work, including the use of the default font is not OK, will be reported

**Comment by amueller** [comment]
I'm not sure what you mean by your last sentence. Can you use the default font that comes with wordcloud?
Does the file exist? I'm not sure about how PIL likes paths to be formatted on windows, but having a double slash after ``C:`` and single slashes everywhere else looks odd.

**Comment by typ431127** [comment]
I do not use the default font

**Comment by typ431127** [comment]
I still use centos7 it

**Comment by amueller** [comment]
Let me rephrase: is there an error if you use the default font?
I'm not sure what you mean with your comment about centos7. Your error is on a windows machine, right?
